.gitignore

```diff
@@ -103,3 +104,66 @@ ENV/
 
 # coverage
 *,cover
+
+
+#intermediate files
+cogframe.h
+lextab.py
+yacctab.py
+virtpython2
+virtpython3
+mediagrains/cogframe.py
+cog.i
+
+# Vagrant
+.vagrant/
+vagrant-support/roles
+
+# Ansible
+*.retry
+
+# HTML versions of RAML docs
+api/*.html
+
+# HTML versions of library docs
+docs/
+
+# pidfiles for local services
+*.pid
+
+# constraints.txt/requirements.txt
+constraints
+constraints.txt
+requirements.txt
+
+# .flake8
+.flake8
+
+# tox.ini
+tox.ini
+
+# .gitignore
+.gitignore
+
+# setup.cfg
+setup.cfg
+
+# commontooling itself
+commontooling
+!static-commontooling/misc/*
+
+# .vscode
+.vscode
+
+# *.gen
+*.gen
+
+# Generated version file
+_version.py
+
+# Temporary directories for files copied out of commontooling
+.tmp/
+
+# Extra gitignore lines from Makefile
+Dockerfile.multi
+Dockerfile.multi
```

Comment:
I didn't notice this before in mediatimestamp, but should these be in here (twice)?

Reply:
Good spot. Sorted.